javascript/javascript1/week2/fullName.js

```diff
@@ -0,0 +1,24 @@
+//Flight booking getFullname function with use of formals:
+function getFullname(firstname, surname, useFormalName, isMistress) {
```

Comment:
Use camelCase for  `firstName` and the function name - `getFullName`
A mistress is a woman who is in a relatively long-term sexual and romantic relationship with someone who is married to a different person.
I'd recommend you `isLady`, `isMadam`.

Reply:
It is done...